.github/actions/e2e/action.yml

```diff
@@ -28,5 +28,5 @@ runs:
 
     - name: Run tests
       working-directory: ${{ inputs.working_directory }}
-      run: pytest -n auto -v
+      run: pytest -n 3 -v
```

Comment:
Each test to be assigned to it's own process.

Reply:
These tests are using our production environment so let's not overwhelm too much and ensure that we do not end up with some rate-limiting issues. In neptune client we used to use `2` as a factor for `pytest` number of processes.

Reply:
With `auto` or `2` processes the problem is that the same process which has already loaded the Neptune Tensorboard integration is used for further testing (which has already been patched and whose wrapped run is already closed) and leads to an error.

Ref: https://github.com/neptune-ai/neptune-tensorboard/actions/runs/5243745411/jobs/9468824502?pr=55#step:4:749

Reply:
Yeah, you're right! We're registering all these hooks globally on the process level and they try to work even after the Run was destroyed and during the evaluation of the next tests. Setting it to `3` probably solves the issue but it doesn't sound like a proper solution 😜 . The easiest one is just to try... catch the `InactiveRun...` exceptions but that sounds really wrong as well. We should probably start to think of some "context manager"-like version of `enable_tensorboard_logging`.

Reply:
Let's ask the Great One.
![Screenshot 2023-06-12 at 23 46 06](https://github.com/neptune-ai/neptune-tensorboard/assets/917619/0099d675-82de-4bd6-a852-858c33f2b9ff)

Reply:
Ohh, I see now. It's double `if` (non-related to each other). I thought it was `if ... elif `. Sorry, my fault.

Reply:
Should we have `context manager` for testing only or actually part of the public API?

Reply:
Why not both 😜 ? Especially that someone may want to log multiple runs (HPO etc.). It's impossible with the current implementation.

Reply:
Have added a context manager version of the API, did you have something similar in mind? Thank you!

Reply:
Updated :)